atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/KvsDeleteBenchmarks.java

```diff
@@ -36,6 +39,18 @@ private Object doDelete(RegeneratingTable<Multimap<Cell, Long>> table) {
         return table.getTableCells();
     }
 
+    private Object doDeleteRange(ConsecutiveNarrowTable table, int numBatches) {
+        Iterable<RangeRequest> rangeRequests;
+        if (numBatches == 1) {
+            rangeRequests = ImmutableList.of(RangeRequest.all());
+        } else {
+            rangeRequests = table.getRangeRequests(1, table.getNumRows() / numBatches);
```

Comment:
This doesn't really actually call getRange, or anything really time-heavy and database hitting, it just generates the requests and uses a tiny smidgen of CPU time and no DB time.